portfolio-project-react/src/components/About/About.jsx

```diff
@@ -12,17 +12,53 @@ import {
 } from "./constants";
 import styles from "./About.module.css";
 
-const About = ({ isEditModeEnabled, totalWorkExperience }) => {
-  const [aboutContent, setAboutContent] = useState(ABOUT_DATA);
-  const [phone, setPhone] = useState(CONTACT_DATA);
-  const [email, setEmail] = useState(EMAIL_DATA);
+const About = ({ totalWorkExperience }) => {
+  const isEditModeEnabled = useContext(IsEditModeEnabled);
+
+  const [aboutContent, setAboutContent] = useState(
+    localStorage.getItem("about") ? localStorage.getItem("about") : ABOUT_DATA
+  );
+
+  const [phone, setPhone] = useState(
+    localStorage.getItem("phone") ? localStorage.getItem("phone") : CONTACT_DATA
+  );
+
+  const [email, setEmail] = useState(
+    localStorage.getItem("email") ? localStorage.getItem("email") : EMAIL_DATA
+  );
+
+  const validateInput = (e) => {
+    const key = e.key;
+
+    const isValidInput = /[\d\s]|Backspace|ArrowLeft|ArrowRight|Delete/i.test(
+      key
+    );
+
+    if (!isValidInput) {
+      e.preventDefault();
+    }
+  };
+
+  useEffect(() => {
+    localStorage.setItem("about", aboutContent);
+  }, [aboutContent]);
+
+  useEffect(() => {
+    localStorage.setItem("phone", phone);
+  }, [phone]);
+
+  useEffect(() => {
+    localStorage.setItem("email", email);
+  }, [email]);
 
   return (
     <div>
       <div className={styles.aboutHeader}>
         <h1 className={styles.titleAbout}>{ABOUT_HEADING}</h1>
         <label>
-          Total Experience: {totalWorkExperience.years} years {totalWorkExperience.months > 0 ? totalWorkExperience.months : 0} months
+          Total Experience: {totalWorkExperience.years} years{" "}
+          {totalWorkExperience.months > 0 ? totalWorkExperience.months : 0}{" "}
```

Comment:
you don't need to show 0 months when user has 0 months experience and the same goes for years if no of years is 0 then only show months